tools/nuget/UWP/Microsoft.MixedReality.WebRTC.UWP.props

```
@@ -0,0 +1,9 @@
+﻿<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Condition="'$(UsingMicrosoftNETSdk)' != 'true'">
```

As per https://github.com/microsoft/MixedReality-WebRTC/pull/634#discussion_r567004196

```suggestion
  <ItemGroup Condition="'$(NuGetProjectStyle)' != 'PackageReference'">
```

---

I applied the suggested change to both `.props` files 👍